cnd/migrations.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+function navigate_to_last_folder() {
+  cd $(ls -d */|tail -n 1)
+}
+
+RELEASE_TAG=$(git describe --tags)
+RELEASE_VERSION=$(echo $RELEASE_TAG | cut -d'-' -f 1)
+echo "Checking for current release version ${RELEASE_VERSION}"
+
+# navigate to latest migration
+cd migrations
+MIGRATIONS_FOLDER=$(pwd)
+navigate_to_last_folder
+LATEST_MIGRATION=$(pwd)
+# get the latest commit of the latest migration
+LATEST_COMMIT=$(git log -n 1 --pretty=format:%H -- $LATEST_MIGRATION)
+
+# check if the commit yields any tags
+TAG_LIST=$(git tag --contains $LATEST_COMMIT)
+
+if [[ $TAG_LIST == *"${RELEASE_VERSION}"* ]]; then
+  echo "A new migration is needed, please enter a name for the new migration based on your planned changes:"
+  echo "Naming according to Diesel conventions, separating words with '_'."
+  read MIGRATION_NAME
+  cd $MIGRATIONS_FOLDER
+  cargo install diesel_cli --no-default-features --features sqlite
```

Comment:
Might want to check whether `diesel` is already installed with:

```
which diesel || cargo install diesel_cli --no-default-features --features sqlite
```

Reply:
we can add this - I feel it does not hurt. it just says `already installed` if it already is.